app/services/session_refresher_test.go

```diff
@@ -41,6 +42,8 @@ func TestSessionRefresher(t *testing.T) {
 		assert.NoError(t, err)
 		assert.NotEmpty(t, identityToken)
 
+		// TODO: parse token and validate SessionID still set - what setup is needed?
```

Comment:
have a look in server/test/asserts.go at AssertIDTokenResponse. i doubt it will be useful in whole, but it should provide an example of parsing and asserting on the JWT.

Reply:
Ah that's nice I was trying to think of how to use the key setup for key store, didn't think to go at it through JWT library directly.  Will give a go hopefully one more force push.  

Need to spend a little time tomorrow to build a docker image locally but the change itself feels very straightforward don't anticipate any problems as long as we deploy the client change with our MFA stuff.

Reply:
Yup that worked perfectly thanks - just force pushed the last test update.